src/spec/ToastMessage.spec.jsx

```diff
@@ -0,0 +1,55 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { describe, it, expect, vi } from "vitest";
+
+import ToastMessage from "../components/Toast/ToastMessage";
+
+describe("Test ToastMessage Component", () => {
+  it("Toast Mesaage should show its props's texts", () => {
+    render(
+      <ToastMessage
+        icon="✅"
+        messages={["passed the test"]}
+        link={"have test"}
+        deleteMessage={() => {}}
+      />,
+    );
+
+    const messages = screen.queryAllByText((content, element) => {
+      return (
+        element.textContent.includes("✅") &&
+        element.textContent.includes("passed the test") &&
+        element.textContent.includes("have test")
+      );
+    });
+
+    expect(messages.length).toBeGreaterThan(0);
```

Comment:
```suggestion
    expect(messages.length).toBe(3);
```

messages 길이가 0보다 크다 보다는
messages 길이가 3이다를 확인하면 더 의미있는 테스트 코드가 될 것 같아요!